### PR TITLE
{bio}[foss/2018b] SKESA v2.9.3, NGS v2.9.3, ncbi-vdb v2.9.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node3101.skitty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6aa0c6e411a94697dea8f94cf58982e8 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2602.swalot.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/3c577dd3dac52a4f45db6c2705b81c25 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.5.1804, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/0809637eedaf67fa7cec664de8ae73dd for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/b13db6be6966d31192776c53201f9571 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/s/SKESA/SKESA-2.9.3-foss-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+name = 'SKESA'
+version = '2.9.3'
+
+homepage = 'https://github.com/ncbi/ngs-tools/tree/master/tools/skesa'
+description = "SKESA is a de-novo sequence read assembler for cultured single isolate genomes based on DeBruijn graphs."
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/ncbi/ngs-tools/archive/']
```
Comment (boegel) [inline easybuild/easyconfigs/s/SKESA/SKESA-2.9.3-foss-2018b.eb]: Download location + version was correct in #7275

**Pull request merged**